singer-io/tap-closeio
# Rewrite of tap

Primarily to:

- Begin syncing new streams
- Bring the tap up-to-date with latest practices

Details:

- Discovery mode is updated to all the latest practices, like usage of `inclusion`
- The JSON schemas have been updated to use `additionalProperties: false` everywhere
- New streams have been added:
  - Custom fields
  - Event log
  - Tasks
  - Users
- tap-tester tests created https://github.com/stitchdata/tap-tester/pull/25
- Project structure has been revamped to split functionality up

To test this I:

- Ran the tap and synced into Redshift (data from a test account)
- Briefly looked through Redshift tables to check structure
- Created and ran tap-tester tests
- Used https://github.com/b-ryan/singer-best-practices-linter
